tests/web/test_web_client_coverage.py

```diff
@@ -8,9 +8,9 @@
 
 
 class TestWebClientCoverage(unittest.TestCase):
-    # 196 endpoints as of May 28, 2020
+    # 200 endpoints as of July 15, 2020
```

Comment:
This should be 205 endpoints. Run the following JavaScript code and add `admin.conversations.whitelist.*` to the ones to be skipped in the `setup` method.

Reply:
Hi Kaz, thank you for the feedback 😄  I added the `admin.conversations.whitelist.*` endpoints to be skipped. I just wanted some quick clarification on the first part as I'm not sure what you mean by the JavaScript code. Also, the current count of the endpoints is based on the current length of the `all_api_methods` list which is 200 as shown in the screenshot below. This number made sense to me as I had added 4 new endpoints to the previous count of 196. Is there something more I should be considering in that count?

![Screen Shot 2020-07-16 at 12 43 34 PM](https://user-images.githubusercontent.com/24510265/87698804-f981a600-c761-11ea-8dac-73d5edac544c.png)

Reply:
@kian2attari If you open https://api.slack.com/methods and run the JS code in dev tool, you'll get the very long comma-separated list value string. Just cut and paste it. Please don't modify the `all_api_methods` manually.

Reply:
Got it! I had completely missed the JS code in the comment before 🤦